brainscore_vision/models/bp_resnet50_julios/setup.py

```diff
@@ -19,7 +19,7 @@
         'Intended Audience :: Developers',
         'License :: OSI Approved :: MIT License',
         'Natural Language :: English',
-        'Programming Language :: Python :: 3.7',
+        'Programming Language :: Python :: 3.11',
```

Comment:
why do plugins specify the python version anyway?

I think we should either:
1. remove the python specification
2. change it as you did if we re-ran plugin tests (including the generic model ones) and they passed
3. leave it as is and just run with the new python version (while acknowledging that they're meant to be run with 3.7)

I won't make the same comment for all other model plugins, but it applies there too

Reply:
Removing python specification